### DISCUSSION TITLE
二次元交流群

### DISCUSSION THREAD

**Discussion by bbdqz** [share121/inter-knot#1977] [其他]
随便聊什么都可以
![698dca303140a288d0367bf859f1d1bb1754845475](https://github.com/user-attachments/assets/eaf6c082-8430-4eb1-a2f8-043522d00f88)
![160984746](https://github.com/user-attachments/assets/32381f5e-91da-4f08-b455-1dab7be7d7f5)
![68747470733a2f2f75706c6f61642d6262732e6d69796f757368652e636f6d2f75706c6f61642f323032332f30332f30392f33313733626333653639666366656265636664613734313334306339616432325f363834313235303331303735363237353637382e706e67](https://github.com/user-attachments/assets/6b09559e-54c4-4a37-991b-e8f92ffe9f35)
![d4da20d6a8f18bdeaa9ba00083dfad4fc6ce89e2](https://github.com/user-attachments/assets/41c47f90-c578-48b8-9c7a-2817c157286a)
应该没有人知道我喜欢芙芙(

**Comment by yileinafs** [comment]
喜欢伊蕾娜

**Comment by YING-ADX** [comment]
我嘞个广州地铁
